client/my-sites/backup/backup-contents-page/file-browser/file-browser-node.tsx

```diff
@@ -165,6 +183,46 @@ const FileBrowserNode: FunctionComponent< FileBrowserNodeProps > = ( {
 		setIsOpen( ! isOpen );
 	}, [ dispatch, isOpen, item, path, setActiveNodePath ] );
 
+	const filterItems = useCallback(
+		( item: FileBrowserItem ) => {
+			// No filter is needed
+			if ( ! applyFiltering ) {
+				return true;
+			}
+			// Check if this node type should always be excluded
+			if ( filterConfig?.excludeTypes?.includes( item.type ) ) {
+				return false;
+			}
+
+			// Check if this node should always be included
+			if ( filterConfig?.alwaysInclude?.includes( item.name ) ) {
```

Comment:
nit/question: it doesn’t impact the current implementation, but I’m wondering if `alwaysInclude` should override `excludeTypes`? For example, should an item that’s explicitly marked as always included still be filtered out if its type is in excludeTypes? Just something to consider depending on the intended behavior.